src/main/java/org/damap/base/rest/DmpDocumentResource.java

```diff
@@ -41,34 +43,74 @@ public class DmpDocumentResource {
    */
   @GET
   @Path("/{dmpId}")
+  @Deprecated(since = "4.3.0")
```

Comment:
To provide more info for other developers (at institutions mostly), if they are using this endpoint. Maybe we can also add an info about removal in the java docs, as it should probably be removed with v5
```suggestion
  @Deprecated(since = "4.3.0", forRemoval = true)
```

Reply:
Done